espn_api/football/fantasy.py

```diff
@@ -0,0 +1,172 @@
+		# print(player.points)
+		if player.points == score and player.onTeamId == team.team_id:
+			return player.name.rsplit(' ', 1)[1] + ', '
+	return ''
+
+# Get team name for a team_id
+def getTeamName(id):
+	return next((y.team_name for y in league.teams if y.team_id == id), None)
+
+def getTeam(name):
+	return next((y for y in league.teams if y.team_name == name), None)
+
+LEAGUE_ID = 306883
+YEAR = 2024
+league = League(LEAGUE_ID, YEAR)
+awards = {}
+players = []
+
+week = 5
+box_scores = league.box_scores(week=week)
+
+week_high = 0
+week_high_team = ''
+week_low = 200
+week_low_team = ''
+week_high_diff = 0
+week_low_diff = 200
+diff_high_team = ''
+diff_low_team = ''
+winners = {}
+losers = {}
+
+# Iterating over matchups
+for matchup in box_scores:
+
+	# Make pile of all players to iterate over 
+	players += matchup.away_lineup + matchup.home_lineup
+
+	# If away team won, add the team to winners, else to losers
+	if matchup.away_score > matchup.home_score:
+		winners[matchup.away_team.team_name] = matchup.away_score
+		losers[matchup.home_team.team_name] = matchup.home_score
+	else:
+		winners[matchup.home_team.team_name] = matchup.home_score
+		losers[matchup.away_team.team_name] = matchup.away_score
+
+	if matchup.away_score < 100:
+		award(matchup.away_team.team_name, 'SUB-100 CLUB')
+	if matchup.home_score < 100:
+		award(matchup.home_team.team_name, 'SUB-100 CLUB')
+
+	# Compute difference between scores
+	temp_week_high_diff = round(abs(matchup.away_score - matchup.home_score))
+	temp_week_low_diff = round(abs(matchup.away_score - matchup.home_score))
+
+	# Computer who won by the most points
+	if temp_week_high_diff > week_high_diff:
+		week_high_diff = temp_week_high_diff
+		diff_high_team = matchup.away_team.team_name if matchup.away_score > matchup.home_score else matchup.home_team.team_name
+		loss_high_team = matchup.away_team.team_name if matchup.away_score < matchup.home_score else matchup.home_team.team_name
+	
+	# Compute who won by the fewest points
+	if temp_week_low_diff < week_low_diff:
+		week_low_diff = temp_week_low_diff
+		diff_low_team = matchup.away_team.team_name if matchup.away_score > matchup.home_score else matchup.home_team.team_name
+		loss_low_team = matchup.away_team.team_name if matchup.away_score < matchup.home_score else matchup.home_team.team_name
+
+	# Compute who had the highest score of the week
+	if matchup.home_score > week_high:
+		week_high = matchup.home_score
+		week_high_team = matchup.home_team.team_name
+	if matchup.away_score > week_high:
+		week_high = matchup.away_score
+		week_high_team = matchup.away_team.team_name
+
+	# Compute who had the lowest score of the week  	
+	if matchup.home_score < week_low:
+		week_low = matchup.home_score
+		week_low_team = matchup.home_team.team_name
+	if matchup.away_score < week_low:
+		week_low = matchup.away_score
+		week_low_team = matchup.away_team.team_name
+
+# Compute lowest scoring winner
+fort_son = min(winners, key=winners.get)
```

Comment:
You could probably do this the same way you do the overall low score of the week